### ISSUE TITLE
Code coverage issue

### ISSUE THREAD

**Issue by kingsleyzissou** [aerogear/offix#329]
## Bug Report

There seems to be an issue with the project code coverage. The code coverage is only being run on the jest config files for each of the packages. The result is that the code coverage is coming back as 100%.

![Screenshot from 2020-01-16 14-23-36](https://user-images.githubusercontent.com/20438192/72532761-dd14b780-386b-11ea-9090-c1cf5ce6c48b.png)

**Comment by darahayes** [comment]
So this issue was introduced when we migrated our tests from `mocha` to `jest` some time ago.

Back then we used `mocha` as a test runner and `nyc` to do code coverage collection. You will notice that nyc is still referenced in our repo.

It turns out that jest has code coverage capabilities built in https://jestjs.io/docs/en/configuration.html#collectcoverage-boolean

Perhaps we just need to enabled that and remove the use of `nyc`. The coverage will probably be output separately per each package but I think that's not necessarily a bad thing.
